.travis.yml

```diff
@@ -11,6 +11,7 @@ cache:
 
 install:
```

Comment:
Don't quite understand your concern here.

There will be no real information there.  Furthermore, we still need to have values for certain test services.